gui/services/forms.py

```diff
@@ -839,9 +839,9 @@ def clean_snmp_community(self):
                 raise forms.ValidationError(_('This field is required.'))
             else:
                 return community
-        if not re.match(r'^[-_,.a-zA-Z0-9\s]+$', community):
+        if not re.match(r'^[-_.a-zA-Z0-9\s]+$', community):
             raise forms.ValidationError(
-                _(u"The community must contain only alphanumeric characters ,"
+                _(u"The community must contain only alphanumeric characters "
```

Comment:
Error message could be more descriptive with regards to what characters are allowed (looks like -, _ and . are allowed in addition to alphanumerics)